boards/riscv/it8xxx2_evb/it8xxx2_evb_defconfig

```diff
@@ -19,7 +19,3 @@ CONFIG_INIT_STACKS=y
 CONFIG_LINKER_ORPHAN_SECTION_PLACE=y
 CONFIG_COMPILER_OPT="-mcmodel=medlow"
 CONFIG_GPIO=y
-# Power Management
-CONFIG_PM=y
-CONFIG_PM_DEVICE=y
-CONFIG_PM_POLICY_CUSTOM=y
```

Comment:
given the custom policy, maybe add this:

```
--- a/boards/riscv/it8xxx2_evb/Kconfig.defconfig
+++ b/boards/riscv/it8xxx2_evb/Kconfig.defconfig
@@ -5,4 +5,13 @@ if BOARD_IT8XXX2_EVB

 config BOARD
        default "it8xxx2_evb"
+
+if PM
+config PM_DEVICE
+       default y
+
+config PM_POLICY_CUSTOM
+       default y
+endif
+
 endif
```

The we need to only enable PM in the app and everything else is set automatically.